lib/index.js

```diff
@@ -153,11 +154,11 @@ allTurtles.push(new Turtle(650, 100, -1.5, 150));
 
 var allLilyPads = [];
 
-allLilyPads.push(new LilyPad(75, 0));
-allLilyPads.push(new LilyPad(175, 0));
-allLilyPads.push(new LilyPad(275, 0));
-allLilyPads.push(new LilyPad(375, 0));
-allLilyPads.push(new LilyPad(475, 0));
+allLilyPads.push(new LilyPad(90, 0));
+allLilyPads.push(new LilyPad(190, 0));
```

Comment:
I wonder if we could step these up by 100 using an algorithm? Might end up being more lines instead of less though.